src/objective-c/tests/Podfile

```diff
@@ -4,11 +4,11 @@ platform :ios, '8.0'
 install! 'cocoapods', :deterministic_uuids => false
 
 def shared_pods
-	pod 'Protobuf', :path => "../../../third_party/protobuf"
-	pod 'BoringSSL', :podspec => ".."
-	pod 'CronetFramework', :podspec => ".."
-	pod 'gRPC', :path => "../../.."
-	pod 'RemoteTest', :path => "RemoteTestClient"
+  pod 'Protobuf', :path => "../../../third_party/protobuf", :inhibit_warnings => true
+  pod 'BoringSSL', :podspec => "..", :inhibit_warnings => true
```

Comment:
Protobuf and BoringSSL currently do not have warnings that could break this test. The silencing flags are added for #5684 says we need to disable this check for them.